### DISCUSSION TITLE
What is the scientific name for a wolf 🐺?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#39] [Galaxy Qs] [answered]
What is the scientific name for a wolf 🐺?

**Comment by sage-etcher** [comment, marked as answer]
Canis lupus

**Reply by Sriansh-raj** [reply]
Congratulations @sage-etcher you just got a `Galaxy Brain` 🎉 

Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃
